src/main/java/namor/template/main/App.java

```diff
@@ -1,9 +1,64 @@
 package namor.template.main;
 
+import java.util.Arrays;
+
 // TODO: 20.03.17 [GUSb] A test class to check everything works, should be deleted with first finished task
 public class App {
 
-    public int add(int a, int b) {
-        return a + b;
+    private int top;
+    int size;
+    int[] stack;
```

Comment:
why stack and size have package default access ? Why not private ? Due to OOP best practices variables should be hidden from outside world by being private